src/sick_tim_common_usb.cpp

```diff
@@ -264,6 +264,7 @@ int SickTimCommonUsb::sendSOPASCommand(const char* request, std::vector<unsigned
 {
   if (device_handle_ == NULL) {
     ROS_ERROR("LIBUSB - device not open");
+    sleep(1);
```

Comment:
@mintar that's true, we should publish the diagnostics before sleeping.  A bad habit, I just looked at the ROS_ERROR.